### ISSUE TITLE
rotate_cam example

### ISSUE THREAD

**Issue by rnixx** [kivy/kivy3#19]
@JohnSumskas hi, what is the https://github.com/kivy/kivy3/blob/development/examples/rotate_cam.py example supposed to demonstrate?

**Comment by JohnSumskas** [comment]
Its meant to be a simple example of the Orbit Control Widget. I.E being able to orbit and move around an object using the right and left click buttons or touch controls.

**Comment by rnixx** [comment]
So i better rename it to orbit.py or similar?

This leads me to another question. The several versions of ``ObjectTrackball`` widgets in the examples. Are these the predecessors of the ``OrbitControlWidget``?

**Comment by JohnSumskas** [comment]
Yes that is correct. But if I recall, Object trackball operated differently. Things got a bit messy on my side when this was a private repo. 

 Yes, I'd agree, renaming it to orbit.py would be more appropriate.

**Comment by rnixx** [comment]
Thanks for the information
